pkg/api/kyverno/v1/types.go

```diff
@@ -56,13 +56,13 @@ type PolicyViolationList struct {
 type Policy struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Spec              Spec         `json:"spec"`
+	Spec              Spec         `json:"spec,omitempty"`
 	Status            PolicyStatus `json:"status"`
 }
 
 // Spec describes policy behavior by its rules
 type Spec struct {
-	Rules                   []Rule `json:"rules"`
+	Rules                   []Rule `json:"rules,omitempty"`
 	ValidationFailureAction string `json:"validationFailureAction"`
 	Background              bool   `json:"background,omitempty"`
```

Comment:
will be better to have this not set as `omitempty` , as the 'false'/disable value will remove the key. But we want to show that it is disabled in background.

Reply:
Removed.